packages/account/package.json

```diff
@@ -38,7 +38,7 @@
         "@deriv/components": "^1.0.0",
         "@deriv/hooks": "^1.0.0",
         "@deriv/integration": "1.0.0",
-        "@deriv/quill-icons": "^1.22.10",
+        "@deriv/quill-icons": "^1.22.12",
```

Comment:
shouldnt we have changes to package-lock.json file with this PR?

Reply:
I think there were no changes to the `package-lock.json` after changing `package.json` and re-building

Reply:
Resolving this conversation for now. Can run `npm run bootstrap:dev` in another PR to update the package-lock since this needs to be merged urgently.

Reply:
Have run `npm run bootstrap:dev` in the current PR and pushed the updated `package-lock.json`